post-policy.go

```diff
@@ -167,6 +167,28 @@ func (p *PostPolicy) SetSuccessStatusAction(status string) error {
 	return nil
 }
 
+// SetUserSpecifiedMetadata - Set user specified metadata as a key/value
+// couple. Can be retrieved through a HEAD request or an event.
+func (p *PostPolicy) SetUserSpecifiedMetadata(key string, value string) error {
```

Comment:
Isn't better to call this SetUserMetadata(), it will be already consistent with other simliar APIs (like SetContentType, etc..)

Reply:
:+1:

Reply:
Sounds good, will push the changes tomorrow

Reply:
You might need to squash your changes and update the PR title here. Thanks @hnb2

Reply:
@harshavardhana Done